test/verify/check-storage-smart

```diff
@@ -61,7 +61,9 @@ class TestStorageSmart(storagelib.StorageSmartCase):
 
         if base_ver > (3, 3, 1):
             return True
-        elif base_ver == (3, 3, 0) and sub_ver >= 3:  # @storage/udisks-daily and Fedora-42
+        elif base_ver == (3, 3, 1) and sub_ver > 1:  # copr @storage/udisks-daily
+            return True
+        elif base_ver == (3, 3, 0) and sub_ver >= 3:
```

Comment:
This is so weird, how does 3.3.0 with a patch release have this fix? Oh well... tests are green.

Reply:
See [libblockdev 3.3.0-3](https://bodhi.fedoraproject.org/updates/FEDORA-2025-059f15a593), the [koji build](https://koji.fedoraproject.org/koji/buildinfo?buildID=2682633) says

> * Thu Mar 20 2025 Tomas Bzatek <tbzatek@redhat.com> - 3.3.0-3
> - smart: Use libatasmart drive self-assessment as an overall status

So that adds up. The only curiousity is that this happened 3 months ago already. Has it been broken for that long?